packages/share/ios/Classes/SharePlugin.m

```diff
@@ -16,8 +16,9 @@ + (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar> *)registrar {
   [shareChannel setMethodCallHandler:^(FlutterMethodCall *call, FlutterResult result) {
     if ([@"share" isEqualToString:call.method]) {
       NSDictionary *arguments = [call arguments];
+      NSString *shareText = arguments[@"text"];
```

Comment:
I might be misunderstanding the nature of this PR. UIActivityViewController's initWithActivityItems expects an NSArray. 

Does the example code not work? Seems to work ok in my testing.

Reply:
@xster it's added to an array inside the `+share:withController:atSource:` class method
https://github.com/flutter/plugins/pull/602/files#diff-95f220dbd86cfc79628c3b434c755122R52

This was existing functionality and has not been modified

Reply:
Ah, you're right. Thanks